demisto_sdk/commands/common/tests/base_validator_test.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from demisto_sdk.commands.common.hook_validations.base_validator import \
+    BaseValidator
+from demisto_sdk.commands.common.hook_validations.structure import \
+    StructureValidator
+from demisto_sdk.tests.constants_test import VALID_INTEGRATION_TEST_PATH
+
+HAS_TESTS_KEY_UNPUTS = [
+    (VALID_INTEGRATION_TEST_PATH, 'integration', True)
+]
+
+
+@pytest.mark.parametrize('file_path, schema, expected', HAS_TESTS_KEY_UNPUTS)
+def test_yml_has_test_key(file_path, schema, expected):
+    structure_validator = StructureValidator(file_path, predefined_scheme=schema)
+    validator = BaseValidator(structure_validator)
+    tests = structure_validator.current_file.get('tests')
+    assert validator.yml_has_test_key(tests, schema) == expected
+
+
+FIND_TEST_MATCH_IMPUT = [
+    ({'integrations': 'integration1', 'playbookID': 'playbook1'}, 'integration1', 'playbook1', True),
+    ({'integrations': 'integration1', 'playbookID': 'playbook1'}, 'integration2', 'playbook1', False),
+    ({'integrations': ['integration1', 'integration2'], 'playbookID': 'playbook1'}, 'integration1', 'playbook1', True),
+    ({'integrations': ['integration1', 'integration2'], 'playbookID': 'playbook1'}, 'integration3', 'playbook1', False),
+    ({'integrations': ['integration1', 'integration2'], 'playbookID': 'playbook1'}, 'integration1', 'playbook2', False),
+]
```

Comment:
These only test the case when the content item in question is an integration. For scripts and playbooks, only the `"playbookID"` key would necessarily be present in the test_config.

Reply:
For some reason all tests are appearing under integrations, even for playbooks and scripts.